### PR TITLE
Executable Tutorial: GitLab implementation of CI/CD 

### PR DESCRIPTION
**Pull request by lazarko**
The idea behind this proposal is to make a wicked tutorial (with Catacoda) on how to implement a CI/CD tool with GitLab.  

**Authors:** 

[Lazar](https://github.com/lazarko)

[Charlotte](https://github.com/charlottea98)

### CODE REVIEW

File: contributions/executable-tutorial/charande-lazarc/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tools for DevOps development. CI is an automated way for continuously merging and testing
+software code, while CD is by continuously make deliveries of software functionality in so-called deployments. 
+The difference between continuous deployment and delivery is that continuous deployment is actually 
+deployed, while continuous delivery is not. 
+
+## Originality
+
+There are no awesome tutorials on Katacoda that illustrates how to create a CI/CD on GitLab. 
+Although, CI/CD are common tools that are used by many software developers, along with many
+types of DevOps tools.
```
Comment (cesarsotovalero) [inline contributions/executable-tutorial/charande-lazarc/README.md]: Can you be more specific. Which tech stack are you going to use for the CI/CD? Which technology specifically?

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]
Hi @lazarko. Can you add more details to your proposal?

File: contributions/executable-tutorial/charande-lazarc/README.md (hunk 1 above)
Comment (lazarko) [reply contributions/executable-tutorial/charande-lazarc/README.md]: @cesarsotovalero I do not understand is it not specified as GitLab? Or is it not counted as Tech Stack?

**Comment by lazarko** [comment]
@cesarsotovalero is it okay now?

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
> @cesarsotovalero is it okay now?

Yes, it is, I going to merge. Good luck!

**Pull request merged**